Dockerfile

```diff
@@ -1,23 +1,33 @@
-FROM python:3.9-slim as build
+FROM node:lts-buster as node-build
```

Comment:
maybe just `node:lts`?

Reply:
`node:lts` equals `node:stretch-lts`, maybe a little old and not smaller than buster version.

BTW, I choose `buster` not `slim-buster` because node compile need python environment.

Reply:
Thanks! Let's use `buster` then